suites/reef/rgw/baremetal/scale_rgw_multi_site.yaml

```diff
@@ -155,6 +155,69 @@ tests:
       name: upload 5M non-current object to versioned bucket
       polarion-id: CEPH-83574428
 
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            cephadm: true
+            commands:
+              - "radosgw-admin sync status"
+              - "radosgw-admin sync group modify --group-id=global_group --status=allowed"
```

Comment:
Post group modify at global level , we need a period update right ?

Reply:
Yes @TejasC88 , updated the pr.